CedarJava/build.gradle

```diff
@@ -87,4 +91,57 @@ test {
         showStandardStreams false
         exceptionFormat 'full'
     }
+}
+
+java {
+    withSourcesJar()
+    withJavadocJar()
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            from components.java
+            groupId = 'com.cedarpolicy'
+            artifactId = 'cedar-java'
+            version = '2.3.3'
+            pom {
+                name = 'CedarJava'
+                description = 'Java bindings for Cedar policy language.'
+                url = 'https://www.cedarpolicy.com'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                developers {
+                    developer {
+                        id = 'cedar'
+                        name = 'Cedar Team'
+                        email = 'cedar-sonatype-team@amazon.com'
+                    }
+                }
+                scm {
+                    connection = 'scm:git:https://github.com/cedar-policy/cedar-java.git'
+                    developerConnection = 'scm:git:https://github.com/cedar-policy/cedar-java.git'
+                    url = 'https://github.com/cedar-policy/cedar-java'
+                }
+            }
+        }
+    }
+    repositories {
+        maven {
+            url = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+            // To publish, uncomment these lines and ensure you have them set in `gradle.properties`
+            // credentials {
```

Comment:
I couldn't figure out how to not evaluate these lines when you run `./gradlew test` so we need to comment them out for CI to pass.